autoload/airline/themes/dark.vim

```diff
@@ -1,37 +1,84 @@
-" Each theme is contained in its own file and declares variables scoped to the
-" file.  These variables represent the possible "modes" that airline can
-" detect.  The mode is the return value of mode(), which gets converted to a
-" readable string.  The following is a list currently supported modes: normal,
-" insert, replace, visual, and inactive.
+" Airline themes are generated based on the following concepts:
+"   * The section of the status line, valid Airline statusline sections are:
+"       * airline_a (left most section)
+"       * airline_b (section just to the right of airline_a)
+"       * airline_c (section just to the right of airline_b)
+"       * airline_x (first section of the right most sections)
+"       * airline_y (section just to the right of airline_x)
+"       * airline_z (right most section)
+"   * The mode of the buffer, as reported by the :mode() function.  Airline 
+"     converts the values reported by mode() to the following:
+"       * normal
+"       * insert
+"       * replace
+"       * visual
+"       * inactive
```

Comment:
`inactive` is not a valid that comes from `mode()`, but is used by airline to style inactive statuslines